portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,14 +47,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = request.getParameter("viewer-comment");
     String commentName = request.getParameter("viewer-comment-name");
-    String commentCombined = "\"" + comment + "\"" + " -" + commentName;
+    String commentCombined = null;
+    if (comment!=null){ //don't create a comment unless there is text in the comment box
+        if (commentName==null){
+            commentName = "Annoymous";
+        }
+     commentCombined = "\"" + comment + "\"" + " -" + commentName;
```

Comment:
Is there a reason to do this vs just having another property on the comment that is the comment name?